client/verta/verta/client.py

```diff
@@ -815,6 +816,8 @@ def get_or_create_registered_model(
             Task type of the registered_model.
         data_type : :mod:`~verta.registry.data_type`, optional
             Data type of the registered_model.
+        pii : :mod:`~verta.registry.pii`, optional
```

Comment:
This is a simple `bool`, so we don't want/need to reference another Python `mod`ule.

Also, I think we can say this is `False` by default—rather than optional—since that's the result in the backend if the user doesn't explicitly pass a value.

```suggestion
        pii : bool, default False
```